code/src/components/Header.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
```

Comment:
In the header we added a  `<hr>` as  a component to get that long line from the design that separates the h1 from the albums